### PR TITLE
refactor: change the interface stored in `_merkleStreamers`

### PR DESCRIPTION
**Pull request by andreivladbrg**
Closes https://github.com/sablier-labs/v2-periphery/issues/185 and addresses feedback left here https://github.com/sablier-labs/v2-periphery/discussions/189

### CODE REVIEW

File: src/interfaces/ISablierV2MerkleStreamerFactory.sol [hunk 1]
```diff
@@ -52,7 +53,7 @@ interface ISablierV2MerkleStreamerFactory {
     /// @param streamDurations The durations for each stream due to the recipient.
     /// @param cancelable Indicates if each stream will be cancelable.
     /// @param ipfsCID Metadata parameter emitted for indexing purposes.
-    /// @param campaignTotalAmount Total amount of ERC-20 assets to be streamed.
+    /// @param aggregateAmount Total amount of ERC-20 assets to be streamed by all recipients.
```
Comment (PaulRBerg) [inline src/interfaces/ISablierV2MerkleStreamerFactory.sol:56]: "to all recipients"

**Review by PaulRBerg — APPROVED** [review]
LGTM, let's just fix the comment below.

I like the "aggregate amount" wording a lot. What would you say about using it in the batch create functions, too? That is, instead of `totalTransferAmount`, we would say `aggregateAmount`.

**Comment by andreivladbrg** [comment]
> What would you say about using it in the batch create functions, too? That is, instead of totalTransferAmount, we would say aggregateAmount.

Both names convey the idea that the variable is a sum of multiple amounts, but I believe that `totalTransferAmount` is more specific and appropriate, it describes what the variable's purpose is in a straightforward manner, the amount is going to be transferred. However, `aggregateAmount` is more general and could be used in a wider range of contexts, like the one we use in the factory contract, only to emit in the event. It doesn't specify what the aggregate amount is for. 

Having said that, I suggest we keep `totalTransferAmount` in the batch create functions. Do you agree?

**Comment by PaulRBerg** [comment]
> `totalTransferAmount` is more specific and appropriate

You may have just gotten used to it. It's not that "specific" and "appropriate" given that it can easily be confounded for each stream's `totalAmount` (notice that the term "total" can be found in both names).

> the amount is going to be transferred

Between the user and the proxy, or between the proxy and Sablier?

> It doesn't specify what the aggregate amount is for.

We will define it in the NatSpec as the sum of all per-stream `totalAmount`s.

> I suggest we keep totalTransferAmount in the batch create functions. Do you agree?

No. But we can debate this disagreement somewhere else. Happy to merge this PR now.

**Review by PaulRBerg — APPROVED** [review]
looking good now

**Pull request merged**